whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -370,10 +370,6 @@ private Response getResponse(HttpServletRequest request, Iterable<RpslObject> re
 
         final RpslObject resultObject = rpslIterator.next();
 
-        if (isIANABlock(resultObject)){
```

Comment:
Why move the check from the service class? Are there object lookups done elsewhere (I can't find the duplication)

Reply:
Hi!
I didn't want to duplicate isIANABlock method in both
I updated setting the method as public and using it from the service instead of throwing the error in the mapping

Reply:
Just a suggestion, make this a util function ? We need to use this in lot of places. Feel free to ignore as it was like this earlier as well